### PR TITLE
Create container factory

### PR DESCRIPTION
**Pull request by gianarb**
Reference #77

@wdalmut

### CODE REVIEW

File: src/App.php [hunk 1]
```diff
@@ -43,38 +43,13 @@ class App
      */
     public function __construct(ContainerInterface $container = null)
     {
-        $this->container = $container ?: static::buildContainer(Loader::load());
-        $container = &$this->container;
+        $this->container = $container ?: Container\PHPDiFactory::buildContainer(Loader::load());
 
         if ($container->has('router') == false) {
```
Comment (wdalmut) [inline src/App.php:51]: The `$container` variable is valid? Or should be `$this->container`?

Comment (gianarb) [inline src/App.php:51]: Tested! :) Now is more readable

File: src/Container/PHPDiFactory.php [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+     *
+     * @return ContainerInterface
+     */
+    public static function buildContainer($config = [])
+    {
+        $builder = new DI\ContainerBuilder();
+        $builder->useAnnotations(true);
+        $builder->addDefinitions(
+            [
+                "event_manager" =>  DI\object('Zend\EventManager\EventManager'),
```
Comment (samsonasik) [inline src/Container/PHPDiFactory.php]: use single quote

File: src/App.php (hunk 1 above)
Comment (wdalmut) [inline src/App.php:51]: yeah comes from the method parameter... sorry miss-read at that time...

**Pull request merged**